libmscore/repeatlist.cpp

```diff
@@ -389,11 +389,11 @@ void RepeatList::collectRepeatListElements()
                         remainder = swap;
                     }
                     // Cross-section of the repeatList
-                    std::list<int> endings = remainder->endings().toStdList();
+                    std::list<int> endings(remainder->endings().begin(), remainder->endings().end());
                     endings.remove_if([&volta](const int& ending) {
                             return !(volta->hasEnding(ending));
                         });
-                    remainder->setEndings(QList<int>::fromStdList(endings));
+                    remainder->setEndings(QList<int>(endings.begin(), endings.end()));
```

Comment:
and this too

Reply:
OK as per https://doc.qt.io/qt-5/qlist.html#fromStdList